### PR TITLE
Add WebAppModuleView and refresh web app node in the beginning

### PR DESCRIPTION
**Pull request by SummerSun**
Each web app has "Deployment Slots" node shown in the UI no matter its slots tree is empty or not. Refresh the web app not to render the node when refreshing all web apps to save user one time of click.

### CODE REVIEW

File: Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/webapp/WebAppModule.java [hunk 1]
```diff
@@ -94,4 +99,26 @@ public void run() {
...
+                resourceEx.getSubscriptionId(), app.id(), app.name(), app.state(), app.defaultHostName(),
+                new HashMap<String, String>() {
+                    {
+                        put("regionName", app.regionName());
+                    }
+                });
+
+            addChildNode(node);
+
+            try {
```
Comment (Eskibear) [inline Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/webapp/WebAppModule.java]: Where comes the exception?

File: Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/webapp/WebAppModuleView.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package com.microsoft.tooling.msservices.serviceexplorer.azure.webapp;
```
Comment (Eskibear) [inline Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/webapp/WebAppModuleView.java]: add copyright

File: Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/webapp/WebAppModulePresenter.java [hunk 3]
```diff
@@ -1,40 +1,19 @@
...
      */
     public void onModuleRefresh() {
-        List<ResourceEx<WebApp>> webapps = new ArrayList<>();
-        webapps.addAll(AzureWebAppMvpModel.getInstance().listAllWebApps(true));
-
-        if (getMvpView() == null) {
-            return;
+        final WebAppModuleView view = getMvpView();
+        if (view != null) {
+            view.renderChildren(AzureWebAppMvpModel.getInstance().listAllWebApps(true));
```
Comment (Eskibear) [inline Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/webapp/WebAppModulePresenter.java]: ` AzureWebAppMvpModel.getInstance().listAllWebApps(true)` should be executed in another thread, in order not to block the UI thread.

File: Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/webapp/WebAppModule.java (hunk 1 above)
Comment (SummerSun) [reply Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/webapp/WebAppModule.java]: Incorrect signature form the `refreshItems()`, will remove it.

**Pull request merged**